tests/test_scm.py

```diff
@@ -24,7 +54,130 @@ def test_commit(self):
         G = Git(self._root_dir)
         G.add(["foo"])
         G.commit("add")
-        self.assertTrue("foo" in self.git.git.ls_files())
+        self.assertIn("foo", self.git.git.ls_files())
+
+    def test_open(self):
+        G = Git(self._root_dir)
+
+        from dvc.utils.compat import open
+
+        # XXX: this is wrong way to deal with encodings, but it
```

Comment:
Is there a reason why this couldn't be extracted into a separate function and used both here and int TestBase above?

Reply:
Moved to Mixin.